docs/fedora-releases-guide/dist-git-onboarding.md

```diff
@@ -146,6 +146,15 @@ to have your package monitored by Upstream Release Monitoring:
 1. make sure there is Fedora mapping for the upstream project in https://release-monitoring.org/projects/
 2. make sure `Monitoring status` on the left side at https://src.fedoraproject.org/rpms/your-package is set to `Monitoring`
 
+:::caution
+
+There could be a variable amount of time between a new upstream tag/release (detected by the release monitoring) and the archive upload.
```

Comment:
```suggestion
There could be a variable amount of time between a new upstream tag/release (detected by the release monitoring) and the archive upload.
Depending on the spec file's `Source` and if the upstream project does not make a release for each tag, you may want to turn on `Check releases instead of tags` option in release-monitoring.
```

Something along those lines